#!/usr/bin/env python3
 
PACKAGE = "control_command_handle"
 
from dynamic_reconfigure.parameter_generator_catkin import *
 
gen = ParameterGenerator()
 

gen.add("Speed", double_t, 0, "Speed of the car", 1.0, 0.0, 20.0)
gen.add("SpeedMod", double_t, 0, "Parameter for modifing the speed of the car", 0.5, 0.0, 10.0)

exit(gen.generate(PACKAGE, "control_command_handle", "control_command_handle_"))
